build/testDetector.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -e
+
+declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
+
+# Load all variables
+source $REPO_DIR/build/__variables.sh
+
+echo
+echo "Building and running tests..."
+testProjectName="Detector.Tests"
+cd "$TESTS_SRC_DIR/$testProjectName"
+dotnet test \
+    --test-adapter-path:. \
+    --logger:"xunit;LogFilePath=$ARTIFACTS_DIR\testResults\\$testProjectName.xml" \
```

Comment:
how about we error out if `ARTIFACTS_DIR` is blank? We should probably document that this is intended for build agents only

Reply:
Sure, good catch